tests/rustdoc/intra-doc/proc-macro-doc.rs

```diff
@@ -0,0 +1,27 @@
+// force-host
+// no-prefer-dynamic
+// compile-flags: --crate-type proc-macro
+
+extern crate proc_macro;
+use proc_macro::*;
+
+// has core/marker/trait.Unpin.html
```

Comment:
I'm not sure what to write here,
```
// @has core/marker/trait.Unpin.html
```
doesn't work and reports
```
8: @has check failed
        File does not exist 'core\\marker\\trait.Unpin.html'
        // @has core/marker/trait.Unpin.html
```
because dependency crates like libcore are not documented in the test suite.

Reply:
You could instead add an aux crate to the test (with `aux-build` and `build-aux-docs` completest headers), and use a link to that instead?  Or else perhaps just make it a `rustdoc-ui` test with `#![deny(rustdoc::broken_intra_doc_links)]` ?

Reply:
Ok, moved to `rustdoc-ui`.